apps/prairielearn/src/schemas/schemas/infoAssessment.json

```diff
@@ -207,15 +194,13 @@
         },
         "role": {
           "description": "DEPRECATED -- do not use.",
-          "deprecated": true,
```

Comment:
I think the loss of the `deprecated` property actually hurts us when it comes to using these as schemas in an editor. If `deprecated` is true, the editor will display that inline:

<img width="203" alt="Screenshot 2025-01-30 at 11 10 46" src="https://github.com/user-attachments/assets/23773044-662d-423c-a436-97fe921284bb" />

Given that, we might want to jump through some hoops to retain these properties, e.g. post-processing after `zod-to-json-schema` to add `deprecated: true` ourselves where appropriate.

Reply:
I can add some simple checks for "DEPRECATED" in the description and adding the deprecated field.

Reply:
Resolved in [b64a47a](https://github.com/PrairieLearn/PrairieLearn/pull/11024/commits/b64a47a03008fdbd89ced37358d9bbe6a12bd38e).